core-it-suite/src/test/java/org/apache/maven/it/IntegrationTestSuite.java

```diff
@@ -190,6 +190,7 @@ public static Test suite()
         suite.addTestSuite( MavenITmng5280SettingsProfilesRepositoriesOrderTest.class );
         suite.addTestSuite( MavenITmng5230MakeReactorWithExcludesTest.class );
         suite.addTestSuite( MavenITmng5224InjectedSettings.class );
+        suite.addTestSuite( MavenITmng5222MojoDeprecatedParamsTest.class );
```

Comment:
This should be as first since we have agreed that the ITs are in reverse chronological of fix date.

Reply:
ok, so sentence "it counts down to zero" it is not always true with - reverse chronological of fix date ... line 90 😄

Reply:
Correct, we had a discussion some time ago. We wanted recently fixed fail fast.